### PR TITLE
Checkov findings

### PR DESCRIPTION
**Pull request by dennisbrouwer91**

### CODE REVIEW

**Review by svashisht03 — APPROVED** [review]
LGTM

File: modules/servicenow/data.tf [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+    "arn:aws:iam::aws:policy/AWSConfigUserAccess",
+    "arn:aws:iam::aws:policy/AWSServiceCatalogAdminReadOnlyAccess"
+  ]
 }
 
-data "aws_iam_policy" "ManagedPolicies" {
-  for_each = local.ManagedPoliciesToGet
-  arn      = each.value
-}
+# data "aws_iam_policy" "ManagedPolicies" {
```
Comment (sbkg0002) [inline modules/servicenow/data.tf:10]: commented? Should be remove then right?

File: modules/servicenow/data.tf [hunk 2]
```diff
@@ -1,13 +1,13 @@
 locals {
-  ManagedPoliciesToGet = {
-    AWSConfigRoleForOrganizations        = "arn:aws:iam::aws:policy/service-role/AWSConfigRoleForOrganizations",
-    AmazonSSMReadOnlyAccess              = "arn:aws:iam::aws:policy/AmazonSSMReadOnlyAccess",
-    AWSConfigUserAccess                  = "arn:aws:iam::aws:policy/AWSConfigUserAccess",
-    AWSServiceCatalogAdminReadOnlyAccess = "arn:aws:iam::aws:policy/AWSServiceCatalogAdminReadOnlyAccess"
-  }
+  ManagedPolicies = [
```
Comment (sbkg0002) [inline modules/servicenow/data.tf:2]: Why the camelcase?

File: modules/servicenow/data.tf (hunk 1 above)
Comment (dennisbrouwer91) [reply modules/servicenow/data.tf:10]: Fixed.

**Review by sbkg0002 — APPROVED** [review]

**Pull request merged**